tests/acceptance/30_generic_methods/permissions_group_acl_present.cf

```diff
@@ -0,0 +1,738 @@
+      "status[24]"    string => "success";
+      "execute[24]"   string => "true";
+
+     # ERROR
+      "files[25]"     string => "fileThatDoesNotExist";
+      "recursive[25]" string => "false";
+      "group[25]"      string => "bin";
+      "ace[25]"       string => "-rw";
+      "create[25]"    string => "false";
+      "mode[25]"      string => "enforce";
+      "status[25]"    string => "error";
+      "execute[25]"   string => "true";
+
+      "files[26]"     string => "fileThatDoesNotExist2";
+      "recursive[26]" string => "true";
+      "group[26]"      string => "bin";
+      "ace[26]"       string => "-rw";
+      "create[26]"    string => "false";
+      "mode[26]"      string => "enforce";
+      "status[26]"    string => "error";
+      "execute[26]"   string => "true";
+
+      "files[27]"     string => "file27";
+      "recursive[27]" string => "false";
+      "group[27]"      string => "aGroupThatDoesNotExist";
+      "ace[27]"       string => "-rw";
+      "create[27]"    string => "true";
+      "initial[27]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:rwx", "user:bin:r", "all:rx" };
+      "exp_lines[27]"  slist => { "user::rwx", "group::r-x", "group:bin:rwx", "user:bin:r--", "other::r-x" };
+      "mode[27]"      string => "enforce";
+      "status[27]"    string => "error";
+      "execute[27]"   string => "true";
+
+    # AUDIT SUCCESS
+     # Copy of success enforce part
+      "files[28]"     string => "file28";
+      "recursive[28]" string => "true";
+      "group[28]"      string => "bin";
+      "ace[28]"       string => "+rwx";
+      "create[28]"    string => "true";
+      "initial[28]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:rwx", "user:bin:r", "all:r" };
+      "exp_lines[28]"  slist => { "user::rwx", "group::r-x", "group:bin:rwx", "user:bin:r--", "other::r--" };
+      "mode[28]"      string => "enforce";
+      "status[28]"    string => "success";
+      "execute[28]"   string => "true";
+
+
+      "files[29]"     string => "file29";
+      "recursive[29]" string => "true";
+      "group[29]"      string => "bin";
+      "ace[29]"       string => "-rw";
+      "create[29]"    string => "true";
+      "initial[29]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:x", "user:bin:r", "all:r" };
+      "exp_lines[29]"  slist => { "user::rwx", "group::r-x", "group:bin:--x", "user:bin:r--", "other::r--" };
+      "mode[29]"      string => "enforce";
+      "status[29]"    string => "success";
+      "execute[29]"   string => "true";
+
+      "files[30]"     string => "file30/.";
+      "recursive[30]" string => "true";
+      "group[30]"      string => "bin";
+      "ace[30]"       string => "-rw";
+      "create[30]"    string => "true";
+      "initial[30]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:-rwx", "user:bin:r", "all:rx" };
+      "exp_lines[30]"  slist => { "user::rwx", "group::r-x", "group:bin:---", "user:bin:r", "other::r-x" };
```

Comment:
hum, this is supposed to be audit, isn't it ?

Reply:
yep! well spotted